### PR TITLE
update Sophos readme

### PR DESCRIPTION
**Pull request by ellis-elastic**
updated the readme file to include links to Sophos's documentation.  Also used the latest product name for Astaro

Type of change

- Enhancement

## What does this PR do?

updated the readme file to include links to Sophos's documentation.  Also used the latest product name for Astaro

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by bmorelli25 — APPROVED** [review]

File: packages/sophos/_dev/build/docs/README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ The Sophos integration collects and parses logs from Sophos Products.
 
 Currently it accepts logs in syslog format or from a file for the following devices:
 
-- `utm` dataset: supports Astaro Security Gateway logs.
-- `xg` dataset: supports Sophos XG SFOS logs.
+- `utm` dataset: supports Unified Threat Management (formally known as Astaro Security Gateway) logs.
```
Comment (r00tu53r) [inline packages/sophos/_dev/build/docs/README.md:7]: Astaro Security Gateway is the older name. So -
```suggestion
- `utm` dataset: supports [Unified Threat Management](https://www.sophos.com/en-us/support/documentation/sophos-utm) (formerly known as Astaro Security Gateway) logs.
```

**Review by r00tu53r — COMMENTED** [review]
Looks good. Just a minor observation. Also there appears to be a version conflict (`changelog.yml and manifest.yml`).

**Review by bmorelli25 — APPROVED** [review]

File: packages/sophos/_dev/build/docs/README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Versions above this are expected to work but have not been tested.
 
 ### Utm log
 
-The `utm` dataset collects Astaro Security Gateway logs.
+The `utm` dataset collects Unified Threat Management (formally known as Astaro Security Gateway) logs.
```
Comment (efd6) [inline packages/sophos/_dev/build/docs/README.md:23]: s/formally/formerly/ (or alternatively drop the parenthetic since it's been noted above).

Comment (bmorelli25) [reply packages/sophos/_dev/build/docs/README.md:23]: Good catch. +1 for removing since it's mentioned earlier.

**Review by efd6 — APPROVED** [review]

**Review by r00tu53r — APPROVED** [review]

**Pull request merged**